x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/HttpContext.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import {
+  borderRadius,
+  colors,
+  fontFamilyCode,
+  px,
+  unit,
+  units
+} from '../../../../../../../style/variables';
+
+import { EuiTitle } from '@elastic/eui';
+import { HttpContext } from '../../../../../../../../typings/Span';
+
+const DatabaseStatement = styled.div`
+  margin-top: ${px(unit)};
+  padding: ${px(units.half)} ${px(unit)};
+  background: ${colors.gray5};
+  border-radius: ${borderRadius};
+  border: 1px solid ${colors.gray4};
+  font-family: ${fontFamilyCode};
+`;
```

Comment:
I think this is duplicated (from [DatabaseContext](https://github.com/elastic/kibana/blob/fa7518fab194c05d95c014902b1e89d1e2de1c79/x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/DatabaseContext.tsx#L34-L41)). I generally prefer having duplication of wrong abstractions - we should just keep it in mind. Preferably there should be an EUI component for this.